src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml

```diff
@@ -401,22 +400,24 @@
                             Style="{StaticResource IconOnlyButtonStyle}"
                             Visibility="{Binding Path=Type, Converter={StaticResource LayoutTypeTemplateToVisibilityConverter}}">
                         <Button.Content>
-                            <TextBlock Text="&#xE8C8;" 
-                                       FontFamily="Segoe MDL2 Assets"
+                            <TextBlock Text="&#xE8C8;"
+                                       FontFamily="{DynamicResource SymbolThemeFontFamily}"
+                                       FontSize="16"
                                        AutomationProperties.Name="{x:Static props:Resources.Create_Custom_From_Template}" />
                         </Button.Content>
                     </Button>
 
-                        <Button Click="DeleteLayout_Click"
-                            x:Name="deleteLayoutButton"
-                            AutomationProperties.Name="{x:Static props:Resources.Delete}"
-                            ToolTip="{x:Static props:Resources.Delete}"   
-                            Style="{StaticResource IconOnlyButtonStyle}"
-                            Visibility="{Binding Path=Type, Converter={StaticResource LayoutTypeCustomToVisibilityConverter}}">
+                    <Button Click="DeleteLayout_Click"
+                        x:Name="deleteLayoutButton"
+                        AutomationProperties.Name="{x:Static props:Resources.Delete}"
+                        ToolTip="{x:Static props:Resources.Delete}"   
+                        Style="{StaticResource IconOnlyButtonStyle}"
+                        Visibility="{Binding Path=Type, Converter={StaticResource LayoutTypeCustomToVisibilityConverter}}">
                         <Button.Content>
-                            <TextBlock Text="&#xE74D;" 
-                                       FontFamily="Segoe MDL2 Assets"
-                                       Foreground="#c50500"
+                            <TextBlock Text="&#xE74D;"
+                                       FontFamily="{DynamicResource SymbolThemeFontFamily}"
+                                       FontSize="16"
+                                       Foreground="Red"
```

Comment:
Good catch - should we use `SystemFillColorCriticalBrush` instead? Hardcoding the value might not meet accessibility requirements when in Dark or HighContrast mode?

Reply:
@SeraphimaZykova idea?

Reply:
Might be a ModernWPF quirk. If it doesn't work, you could consider adding the right brushes to the light/dark/highcontrast resource dictionaries manually. But that's some work.

If not, I'd revert back to `#c50500` as that color wasn't flagged during the accessibility reviews - I know using the default red color was in the past.